FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -0,0 +1,2119 @@
+		<fibo-fnd-utl-av:commonDesignation>DIVIDE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DVD</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DV</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Drive">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>drive</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DR</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRIV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRIVE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>DRV</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DR</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Drives">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>drives</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>DRIVES</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>DRS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Estate">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>estate</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>EST</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>ESTATE</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>EST</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Estates">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>estates</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>ESTATES</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>ESTS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>ESTS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Expressway">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>expressway</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>EXP</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>EXPR</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>EXPRESS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>EXPRESSWAY</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>EXPW</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>EXPY</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>EXPY</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Extension">
```

Comment:
Likely to break the unique names rule

Reply:
Same comment as for estate - these are individuals that are part of the addressing standard for the US, and the namespace should be used to distinguish them.